lib/ipfsbstore/ipfsbstore.go

```diff
@@ -0,0 +1,106 @@
+package ipfsbstore
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+
+	"golang.org/x/xerrors"
+
+	"github.com/multiformats/go-multihash"
+
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	httpapi "github.com/ipfs/go-ipfs-http-client"
+	iface "github.com/ipfs/interface-go-ipfs-core"
+	"github.com/ipfs/interface-go-ipfs-core/options"
+	"github.com/ipfs/interface-go-ipfs-core/path"
+)
+
+type IpfsBstore struct {
+	ctx context.Context
+	api iface.CoreAPI
+}
+
+func NewIpfsBstore(ctx context.Context) (*IpfsBstore, error) {
```

Comment:
Maybe allow people to configure the api endpoint for the daemon?

Reply:
You can with an env var: https://github.com/ipfs/go-ipfs-http-client/blob/master/api.go#L48

We may want to add a config field for this at some point, but an env var is how you tell ipfs where it's repo lives, so most people with custom location will have it set already